searchlib/src/vespa/searchlib/attribute/reference_attribute.h

```diff
@@ -29,32 +30,50 @@ class ReferenceAttribute : public NotImplementedAttribute
     using GlobalId = document::GlobalId;
     class Reference {
         GlobalId _gid;
-        mutable uint32_t _lid;
+        mutable uint32_t _lid;  // referenced lid
+        mutable EntryRef _pidx; // map from gid to lids referencing gid
```

Comment:
Consider renaming '_pidx' -> '_revMapIdx' for clarity. Also rename local 'pidx' variables -> 'revMapIdx'.